resources/js/components/Cropper.vue

```diff
@@ -26,7 +26,7 @@
       image: Object,
       configs: {
         type: Object,
-        default: {},
+        default: => {},
```

Comment:
Your code will not compile. I added this myself to the code. New release 2.7.2 containing the validation fix will be drafted.